packages/admin/src/assets/tocco_white.svg

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ist a new file necessary?  Usually the color of a svg can be changed and of course the orientation.